docs/fuzz-targets.md

```diff
@@ -188,7 +188,7 @@ Alternatively, you can add a new script to your package.json:
 
 ```json
 "scripts": {
-	"coverage": "jazzer -d -i target -i another_target -e nothing <fuzzer parameters> --corpus <corpus directories> --coverage -- <libFuzzer parameters>"
+	"coverage": "jazzer -d -i target -i another_target <fuzzer parameters> --corpus <corpus directories> --coverage -- <libFuzzer parameters>"
```

Comment:
Can we mention somewhere that if `instrumentation_includes` or `instrumentation_excludes` is set, the default of the other one is not used anymore? BTW wrong flag name in the text below.

Reply:
good point